### ISSUE TITLE
Multiple Phpstorm issues

### ISSUE THREAD

**Issue by LordMonoxide** [microsoft/wslg#214]
# Environment

```none
Windows build number: 10.0.21382.1
Your Distribution version: Ubuntu 20.04
Your WSLg version: 1.0.19
```

# Steps to reproduce
There are a few issues I've noticed:
1. Open a few editor windows - you should be able to click and drag to reorder the editor tabs but can't
2. Menus (File, Edit, etc.) do not show the hover effect on mouse over. When you click on them, the menu appears lower and to the right of where it should, and immediately closes when you release the left mouse button, unless you first move the mouse over to where the menu opened. Releasing the mouse button on a menu item will act as if it was clicked. (image attachment 1)
3. There are various places where the mouse hover position does not match the cursor position. (image attachment 2: I'm actually hovering over "logs"). If I click it selects the correct item, even though the hover is offset. Strangely, this doesn't apply to all hovers. Hovering over editor tabs and buttons highlights them correctly. Hovering over code to view the documentation popup works, but the window pops up too far down and right like the menus.
4. The right click context menu in code windows pops up with the same offset as the other menus, and when clicking off of it to make it disappear, it places the text cursor up and to the left of the mouse cursor instead of where you clicked.

**WSL logs**: 

* [pulseaudio.log](https://github.com/microsoft/wslg/files/6509509/pulseaudio.log)
* [weston.log](https://github.com/microsoft/wslg/files/6509512/weston.log)
* [versions.txt](https://github.com/microsoft/wslg/files/6509522/versions.txt)

# Screenshots

![image](https://user-images.githubusercontent.com/437657/118828273-7f471400-b893-11eb-993b-b1bb383d2fd3.png)

![image](https://user-images.githubusercontent.com/437657/118828739-ea90e600-b893-11eb-963d-6e5ea0823d7d.png)

**Comment by NyxCode** [comment]
I too noticed this issue in CLion.
However, for me, this only occurs when the windows is maximized. 
![img](https://i.imgur.com/USLhV5s.png)

**Comment by LordMonoxide** [comment]
I just tested that and it does indeed work when not maximized, and even continues to work correctly once maximized again

**Comment by nelsonjchen** [comment]
Find in path has an annoying issue where the Window simply doesn't work most of the time. I press Ctrl Shift F and the window flashes open and closes immediately usually. The worst part is that I can't find a reliable workaround either. If I right click and use Find in Path from Project, the same issue still happens. 

I'm not sure why Find in Path is having issues when the window from Shift-Shift still works.

**Comment by nlundquist** [comment]
I was noticing similar issues in WebStorm, in particular not being able to press "enter" in the terminal tool window. Strangely, when launching WebStorm from a WSL terminal rather than by running `C:\WINDOWS\System32\wslg.exe ~ -d Ubuntu webstorm` , this issue doesn't occur. I'm able to reproduce this effect consistently.

**Comment by LordMonoxide** [comment]
> Strangely, when launching WebStorm from a WSL terminal rather than by running `C:\WINDOWS\System32\wslg.exe ~ -d Ubuntu webstorm` , this issue doesn't occur.

I launch everything from the terminal, so I don't think it's strictly related to how it's launched

**Comment by LordMonoxide** [comment]
Since upgrading to Windows 11, the offset mouse input no longer seems to be happening for me. The "Find in files" dialog is still very unstable though.

**Comment by acirulis** [comment]
Just updated to Windows 11 (22000.100) and installed Ubuntu 20.04 with WSL2g. PhpStorm 212.4746.100 is basically unusable:
- "Find in files" keeps closing
- Window always starts in non-maximized state.
- When maximized - menus does not work correctly (have to hold mouse button for hover to work)
- ENTER key does not work in Terminal (when app is started from windows shortcut wsl /home/.../phpstorm.sh). Works when launched from WSL2 directly.
- JetBrains Toolbox app does not seem to work either.

Tried also Debian distro, without any changes. I have never had such problems with PhpStorm on Windows, native linux or even on WSL2 with VcXsrv.
Still having high hopes for WSL2g so lets wait for any updates.

**Comment by stevenobird** [comment]
> @stevenobird Hello.
> I'm sorry for worry, but can you help with getting wsl components versions? I try to run `wsl --version`, but this utility has no this option. May be I should to update wsl?

Yes, installing said app from the Microsoft Store is necessary for that. Then you should be able to update WSL with `wsl --update`.

**Comment by speller** [comment]
I have an issue that window contents are not redrawn until resize. When I run the IDE, it shows a blank gray window. When I resize it, it renders well. If a dialog window pops up, its window is also blank until resizing. The search dialog window doesn't appear at all. It tries but then disappears immediately.

**Comment by mmalack** [comment]
>     * "Find in files" keeps closing

For the time being, I have found a workaround for the "Find in files" window that always closes.

The prerequisite is that it remains open at least once. It only worked for me directly after starting PhpStorm.
Then you have to pin the window in the upper right corner and can leave it with Esc. Since then, the window always stays open.

Maybe this helps someone. In any case, this was the bug that annoyed me the most...

**Comment by ansien** [comment]
Same problems happening here.

W11 - build 22000.493
Ubuntu 20.04.3 LTS

**Comment by JohnRusk** [comment]
Some reports of improvements in 2022 releases here https://youtrack.jetbrains.com/issue/IDEA-265390

**Comment by choarau-craft** [comment]
I'm still having the same issue, can't find a way to open a new window not in full screen
